godtools/App/Share/SwiftUI Views/RoundedCorner.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  RoundedCorner.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 4/11/22.
+//  Copyright © 2022 Cru. All rights reserved.
+//
+
+import SwiftUI
+
+/// For use in the custom .cornerRadius modifier
```

Comment:
Agreed on the comment.  As far as moving the Bezier path logic into the extension, I'm not sure I understand how that would work since the `clipShape` method requires a Shape, not just the path.  Or is there a way around it that I'm missing?

Reply:
Hey @rachaelblue I see what you're saying, Shape is a protocol so we will need some type of concrete struct or class. This works perfect for that.